.circle/install-mongo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+sudo apt-get purge mongodb-org*
```

Comment:
Can't we use the similar ramdisk approach as we used on Travis?

It speeds things up and the script also allows us to install and run multiple and specific versions of MongoDB.

> -  - if [ ${TASK} = 'unit' ] || [ ${TASK} = 'integration' ] || [ ${TASK} = 'packs-tests' ]; then sudo -E ./scripts/travis/create-and-mount-ramdisk.sh; fi
> -  - if [ ${TASK} = 'unit' ] || [ ${TASK} = 'integration' ] || [ ${TASK} = 'packs-tests' ]; then ./scripts/travis/install-and-run-mongodb.sh; fi